pkg/propertyprovider/aks/trackers/pods.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package trackers
+
+import (
+	"fmt"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// PodTracker helps track specific stats about pods in a Kubernetes cluster, e.g., the sum
+// of its requested resources.
+type PodTracker struct {
+	totalRequested corev1.ResourceList
```

Comment:
add a comment to indicate we'll ignore the container/pod if requested not set?

Reply:
The cost info part has already included the formulas on that front. As for the pod tracker, I don't we will be ignoring the pod/container per se; the pod/container literally requested zero resource

Reply:
makes sense